Makefile

```diff
@@ -54,10 +56,23 @@ warmup: ## Warmup the cache
 purge: ## Purge cache and logs
 	rm -rf var/cache/* var/logs/*
 
+## —— Doctrine ▶️ ———————————————————————————————————————————————————————————————
+dgm: ## Generate a blank migration class
+	$(SYMFONY) doctrine:migrations:generate
+dmm: ## Execute migrations
+	$(SYMFONY) doctrine:migrations:migrate
```

Comment:
```suggestion
	$(SYMFONY) d:m:m
```

Reply:
I wanted to be explicit on Makefile. Shorter command is nice when you must to type it.